### PR TITLE
add kafka schema JSON support

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: internal/service/kafka/resource_kafka_schema.go [hunk 1]
```diff
@@ -138,13 +147,13 @@ func resourceKafkaSchemaCreate(ctx context.Context, d *schema.ResourceData, m in
...
 		if err != nil {
-			return diag.FromErr(err)
+			return diag.FromErr(fmt.Errorf("unable to update configuration: %w", err))
 		}
 	}
 
 	version, err := kafkaSchemaSubjectGetLastVersion(m, project, serviceName, subjectName)
 	if err != nil {
-		return diag.FromErr(err)
+		return diag.FromErr(fmt.Errorf("unable to to get last version: %w", err))
```
Comment (dmitrii-vasilev) [inline internal/service/kafka/resource_kafka_schema.go:156]: Too many `to`'s

**Review by dmitrii-vasilev — DISMISSED** [review]
Looks good. Just a small typo in the error message

File: internal/service/kafka/resource_kafka_schema.go (hunk 1 above)
Comment (ivan-savciuc) [reply internal/service/kafka/resource_kafka_schema.go]: fixed

**Comment by ivan-savciuc** [comment]
> minor nit: formatting of the json data in the tests looks off

yes indeed, but this is how `make testfmt` likes it to be, if I format it correctly we will get `make lint` CI errors. I suggest lets keep it as it is and I will create a ticket to fix `testfmt` JSON formating

**Review by dmitrii-vasilev — APPROVED** [review]
LGTM

**Pull request merged**